src/vm/arm64/asmhelpers.S

```diff
@@ -562,7 +562,7 @@ LEAF_END SinglecastDelegateInvokeStub, _TEXT
 #ifdef FEATURE_COMINTEROP
 
 #define ComCallPreStub_FrameSize (SIZEOF__GSCookie + SIZEOF__ComMethodFrame)
-#define ComCallPreStub_FirstStackAdjust  (SIZEOF__ArgumentRegisters + 2 * 8) // reg args , fp & lr already pushed
+#define ComCallPreStub_FirstStackAdjust  (8 + SIZEOF__ArgumentRegisters + 2 * 8) // x8, reg args , fp & lr already pushed
```

Comment:
You might want to consider changing this to `(8 + SIZEOF__ArgumentRegisters + (2 * 8))` in case someone gets clever and decides to "optimize" and converts it to `(SIZEOF__ArgumentRegisters + 10 * 8)` 😞